lib/commands/create_project/update_application_id_command.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:bond_cli/core/tasks/create_project/android/tasks/update_application_id_task.dart';
+import 'package:bond_cli/core/utils.dart';
+import 'package:bond_cli/core/validators.dart';
+import 'package:native_project_manipulator/platforms/android/android.dart';
+
+class UpdateApplicationIdCommand extends Command<void> {
+  @override
+  String get name => 'android_app_id';
+
+  @override
+  String get description => 'Update the Android application ID.';
+
+  UpdateApplicationIdCommand() {
+    argParser.addOption('applicationId',
+        abbr: 'a', help: 'Specify the new Android application ID.');
+  }
+
+  @override
+  void run() async {
+    var applicationId = argResults?['applicationId'] as String?;
+
+    applicationId ??= XInput.askValue(
+      'Enter New Android Application ID:',
+      null,
+      validators: [
+        NonEmptyValidator(),
```

Comment:
i think there are a customer validator for bundle id and applications id

Reply:
ok